mitosheet/src/components/taskpanes/Graph/GraphSidebar.tsx

```diff
@@ -112,17 +125,40 @@ const GraphSidebar = (props: {
     sheetDataArray: SheetData[];
     columnIDsMapArray: ColumnIDsMap[],
     dfNames: string[];
-    graphSidebarSheet: number;
+    graphID: GraphID
     columnDtypesMap: Record<string, string>;
     mitoAPI: MitoAPI;
     setUIState: React.Dispatch<React.SetStateAction<UIState>>;
+    uiState: UIState;
     graphDataJSON: GraphDataJSON
     lastStepIndex: number
 }): JSX.Element => {
 
+    /*
+        The graphID is the keystone of the graphSidebar. Each graph tab has one graphID that does not switch even if the user changes source data sheets. 
+        
+        In order to properly open a graph in Mito, there are a few things that need to occur:
+            1. We need to update the uiState's `selectedTabType` to "graph" so that the footer selects the correct tab
+            2. We need to set the uiState's `currTaskpaneOpen` to "graph" so that we actually display the graph
```

Comment:
If these two are true, can we potentially simplify and not use `selectedTabType`? We can just check if `currTaskpaneOpen` is `graph` - and otherwise it's a sheet tab selected?

Would this improve things? What do you think?

It's a bit of a red flag that you have to remember to update _two_ pieces of the UI state to pretty much the same variable - and that they always move in unison!

Reply:
As per our discussion, we're not going to separate these